CONTRIBUTING.md

```diff
@@ -1,47 +1,73 @@
 # Contribution Guidelines
 
-Here you will find the contribution guidelines.
-
-<details>
-<summary><strong><em>Click here to see the Table of Contents</em></strong></summary>
+Contribution are welcome! Here's a few things to know:
 
 * [Microsoft Contributor License Agreement](#microsoft-contributor-license-agreement)
-* [Recommenders Team Contribution Guidelines](#recommenders-team-contribution-guidelines)
-  * [Test Driven Development](#test-driven-development)
-  * [Python and Docstrings Style](#python-and-docstrings-style)
+* [Steps to Contributing](#steps-to-contributing)
+* [Coding Guidelines](#recommenders-team-contribution-guidelines)
 * [Code of Conduct](#code-of-conduct)
-</details>
 
 ## Microsoft Contributor License Agreement
 
 Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-## Recommenders Team Contribution Guidelines
+## Steps to Contributing
 
-#### Test Driven Development
+Here are the basic steps to get started with your first contribution. Please reach out with any questions.
+1. Use [open issues](https://github.com/Microsoft/Recommenders/issues) to discuss the proposed changes (create an issue describing changes if necessary to collect feedback).
```

Comment:
Maybe worth mentioning that it is good practice to tag the issue with the provided labels, which makes easy for users to retrieve issues of interest.

Reply:
👍